docs/markdown/Quick-guide.md

```diff
@@ -1,44 +1,84 @@
 ---
-title: Quick guide
-short-description: Guide to get started using meson
+title: Quickstart Guide
+short-description: Getting Started using mesonbuild
 ...
 
 # Using Meson
 
-Meson has been designed to be as easy to use as possible. This page
-outlines the basic use cases. For more advanced cases refer to Meson's
-command line help which is accessible with the command `meson --help`.
+Meson has been designed to be as simple to use as possible. This page
+outlines the initial steps needed for installation, troubleshooting,
+and standard use. 
 
-Requirements
---
+For more advanced configuration please refer to the command line help `meson --help` 
+or the meson documentation located at the [mesonbuild](https://mesonbuild.com) website.
+
+Table of Contents:
+* [Requirements](#requirements)
+* [Installation using package manager](#installation-using-package-manager)
+* [Installation using python](#installation-using-python)
+* [Installation from source](#installation-from-source)
+* [Troubleshooting](#troubleshooting)
+* [Compiling a meson project](#compiling-a-meson-project)
+* [Using meson as a distro packager](#using-meson-as-a-distro-packager)
 
-Meson has two main dependencies.
+Requirements:
+--
 
 * [Python 3](https://python.org)
 * [Ninja](https://github.com/ninja-build/ninja/)
 
-Ninja is only needed if you use the Ninja backend. Meson can also
-generate native VS and XCode project files.
+*Ninja is only needed if you use the Ninja backend. Meson can also
+generate native VS and XCode project files.*
+
+
+Installation using package manager
+--
 
-On Ubuntu these can be easily installed with the following command:
+Ubuntu:
 
 ```console
 $ sudo apt-get install python3 python3-pip python3-setuptools \
                        python3-wheel ninja-build
 ```
+*Due to our frequent release cycle and development speed, distro packaged software may quickly become outdated.*  
 
-The best way to get Meson is to `pip install` it for your user
+Installation using python
+--
+Requirements: **pip3** 
+
+The best way to receive the most up-to-date version of mesonbuild. 
 
+Install as a local user:
 ```console
 $ pip3 install --user meson
 ```
 
-You can also use Meson as packaged by your distro, but beware that due
-to our frequent release cycle and development speed this version might
-be out of date.
+Install as root:
```

Comment:
This is poor practice to override your package manager, and I don't think it should be encouraged. Those who know to do this will know how to do it without causing trouble, but those who don't and just find out about it here won't, and could break something.

Reply:
Yes, we should encourage people to use `--user` instead.

Reply:
I'm unclear on what specific change here would be needed so I haven't included it in the recent commit. 
It may be bad practice but I think its absence might be a little clunky and it wouldn't address the issue of what would you do if after the user install doesn't run properly.

Reply:
I've added the proposed changes.